docs/cli-reference/dfx-deps.mdx

```diff
@@ -30,7 +30,7 @@ dfx deps pull --help
 
 ## dfx deps pull
 
-Use the `dfx deps pull` command to pull dependencies as defined in `dfx.json`.
+Use the `dfx deps pull` command to pull dependencies as defined in `dfx.json`. It will resolve all indirect dependencies.
```

Comment:
```suggestion
Use the `dfx deps pull` command to pull dependencies as defined in `dfx.json`. It will resolve all transitive dependencies.
```

AFAIK those are usually called `transitive`